### ISSUE TITLE
ResourceType.GRANT_ON_ALL missing from change is a grant execution_strategy_for_change

### ISSUE THREAD

**Issue by toadies** [Titan-Systems/titan#80]
`apply` is trying to created an `grant ownership` sql for GrantOnAll resources.

I was able to circumvent this by including resource type GRANT_ON_ALL to the function `execution_strategy_for_change`

```
def execution_strategy_for_change(
    change: ResourceChange,
    usable_roles: list[str],
    default_role: str,
    role_privileges: dict[str, list[dict]],
) -> tuple[str, bool]:

    change_is_a_grant = change.urn.resource_type in (
        ResourceType.GRANT,
        ResourceType.FUTURE_GRANT,
        ResourceType.ROLE_GRANT,
        ResourceType.GRANT_ON_ALL
    )
```

**Comment by teej** [comment]
Fixed in [v0.8.6](https://github.com/Titan-Systems/titan/releases/tag/v0.8.6)

**Comment by toadies** [comment]
Thank you for fixing and creating such an awesome project.
